src/Rules/StarterWeb/AI/IndividualAuth/project.json

```diff
@@ -51,8 +55,13 @@ $endif$
   "tools": {$if$ ($context$ == WebCore)
     "dotnet-aspnet-codegenerator": "1.0.0-*",
     "dotnet-ef": "1.0.0-*",$endif$
-    "Microsoft.AspNetCore.Server.IISIntegration.Tools": "1.0.0-*",
-    "dotnet-razor-tooling": "1.0.0-*"
+    "Microsoft.AspNetCore.Server.IISIntegration.Tools": "1.0.0-*"$if$ ($context$ == WebCore),
+    "dotnet-razor-tooling": {
+      "version": "__AspNetCorePlatformVersion__",
+      "imports": [
```

Comment:
Since we only need 1 import for razor-tooling lets just do the `"imports": "..."` syntax, same elsewhere.